build/entrypoint.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/sh
 
 echo "Building ${RAILS_ENV}"
+sudo chown -R app:app /data/log
```

Comment:
If we need this for development, we should probably introduce a dev entrypoint script.  We definitely shouldn't do this in production since it forces the existence of sudo (see prior note)